content/community/code.md

```diff
@@ -0,0 +1,31 @@
+# Community Code
+
+Here is a curated list of community created repositories, libraries, components, projects, and code that use Pulumi. If you believe something is missing, send us a [pull request](https://github.com/pulumi/docs/edit/master/content/community/code.md). 
```

Comment:
could we reframe the second sentence to invite people to add things instead of if they think something is missing?